onnx/defs/math/defs.cc

```diff
@@ -1168,4 +1168,33 @@ ONNX_OPERATOR_SET_SCHEMA(
             {"tensor(float16)", "tensor(float)", "tensor(double)"},
             "Constrain input and output types to float tensors.")
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
+
+
+static const char* Sign_ver9_doc = R"DOC(
+lalala
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+	Sign,
+	9,
+	OpSchema()
+		.SetDoc(Sign_ver9_doc)
+		.Input(0, "input", "Input tensor", "T")
+		.Output(
+			0,
+			"output",
+			"The sign of the input tensor "
+			"computed element-wise",
+			"T")
```

Comment:
The output type should be int32

Reply:
I think it is not neccesary to be a int32? Or we can set an optional attribute?

Reply:
I think output tensor should have the same type as the input tensor. At least both tensorflow and pytorch follow this rule.

Reply:
Okay, then the shape inference function below is incorrect. I was going by that.

Reply:
@gramalingam Wonder why it is incorrect? The output here should use the same shape&type as the input...

Reply:
But it says "updateOutputElemType(ctx, 0, TensorProto::INT32);". I think the whole thing can be replaced by .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput).

Reply:
oh damn I didnt push this. Thanks!